codec-http/src/main/java/io/netty/handler/codec/http/HttpHeaderValidationUtil.java

```diff
@@ -161,138 +161,4 @@ private static int verifyValidHeaderValueCharSequence(CharSequence value) {
         }
         return -1;
     }
-
-    /**
-     * Validate a <a href="https://tools.ietf.org/html/rfc7230#section-3.2.6">token</a> contains only allowed
-     * characters.
-     * <p>
-     * The <a href="https://tools.ietf.org/html/rfc2616#section-2.2">token</a> format is used for variety of HTTP
-     * components, like  <a href="https://tools.ietf.org/html/rfc6265#section-4.1.1">cookie-name</a>,
-     * <a href="https://tools.ietf.org/html/rfc7230#section-3.2.6">field-name</a> of a
-     * <a href="https://tools.ietf.org/html/rfc7230#section-3.2">header-field</a>, or
-     * <a href="https://tools.ietf.org/html/rfc7231#section-4">request method</a>.
-     *
-     * @param token the token to validate.
-     * @return the index of the first invalid token character found, or {@code -1} if there are none.
-     */
-    public static int validateToken(CharSequence token) {
```

Comment:
You should keep the public methods and have them delegate to `HttpUtil` as otherwise this is a breaking change.

Reply:
Sorry, I’m making some adjustments and may have caused you some trouble.